spec/hardware/bootdisk.fmf

```diff
@@ -0,0 +1,36 @@
+summary: |
+    Select a guest with a specific boot-disk.
```

Comment:
"Select or create ..." - with OpenStack, bootdisk can affect the VM creation, maybe testcloud would be able to deliver as well.